NOTICE.txt

```diff
@@ -7,17 +7,247 @@ NOTICES:
 This document includes a list of open source components used in Mattermost Mobile, including those that have been modified.
 
 --------
+## analytics-react-native
+
+This product contains 'analytics-react-native' by Javier Alvarez.
+
+A React Native client for [Segment](https://segment.com) — The hassle-free way to integrate analytics into any application.
+
+* HOMEPAGE:
+  * https://github.com/neiker/analytics-react-native
+
+* LICENSE: MIT
+
+MIT License
+
+Copyright (c) 2016 Javier Alvarez
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+---
+
+## commonmark
+
+This product contains 'commonmark' by John MacFarlane.
```

Comment:
a modified version of 'commonmark'

Reply:
I'm happy to update the automation to distinguish "a modified version of..." from the unmodified upstream, where possible.

Would it be sufficient to look at the `package.json` version constraint, and if we find a version like `"github:xxxx"`, where xxxx is not the official upstream repo, then we say it's "modified"?

Reply:
done, the PR is now updated with "a modified version of..." as discussed above.